### PR TITLE
{lib}[GCCcore/10.2.0] re2c v2.0.3, PCRE2 v10.35, double-conversion v3.1.5, ...

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 8 out of 13 (13 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/4f20dde3c902202869a820d20538a037 for a full test report.

**Comment by Micket** [comment]
@branfosj I dropped a patch from NSS when i updated it. The patched lines no longer existed. Can you verify if this works on power?

File: easybuild/easyconfigs/n/NASM/NASM-2.15.05-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+homepage = 'https://www.nasm.us/'
+
+description = """NASM: General-purpose x86 assembler"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://www.nasm.us/pub/nasm/releasebuilds/%(version)s']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['34fd26c70a277a9fdd54cb5ecf389badedaf48047b269d1008fbc819b24e80bc']
```
Comment (branfosj) [inline easybuild/easyconfigs/n/NASM/NASM-2.15.05-GCCcore-10.2.0.eb:26]:
```suggestion
checksums = ['3c4b8339e5ab54b1bcb2316101f8985a5da50a3f9e504d43fa6f35668bee2fd0']
```
The 34fd... checksums if for NASM 2.14.02.

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 11 out of 12 (12 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/62b5f24435f1ba0e827131cca8e0c2d9 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 12 out of 12 (12 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/a925c139f6480e0d35c8ebcdefc3b265 for a full test report.

**Comment by branfosj** [comment]
> @branfosj I dropped a patch from NSS when i updated it. The patched lines no longer existed. Can you verify if this works on power?

Yes, that looks fine. A test report will arrive in a while.

I've also built `NSS-3.57-GCCcore-10.2.0.eb` including running the NSS tests, by adding:
```
buildopts += " && cd %(builddir)s/%(namelower)s-%(version)s/%(namelower)s/tests && BUILD_OPT=1 USE_64=1 ./all.sh "
```
All the tests passed.

**Comment by Micket** [comment]
Ugh. I accidentally pressed edit instead of reply.. I'm tired, sorry @branfosj , i don't know if i can restore your comment

**Comment by branfosj** [comment]
I've reverted the comment.

Those tests take about an hour to run - so I do not think that makes sense to always run. We could possibly have it as a comment that people are made aware that they exist.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 12 out of 12 (12 easyconfigs in total)
bear-pg0305u03a.bear.cluster - Linux RHEL 7.6, POWER, 8335-GTX (power9le), Python 3.6.8
See https://gist.github.com/cf24b01f7701ae1a08bb1d8915cae530 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/c819d35a7084f9360108d57689f5e8e5 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 12 out of 12 (12 easyconfigs in total)
freja - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz (skylake), Python 3.8.5
See https://gist.github.com/fad2a91dbbd9d14f2a7f6fc7b6a56563 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @Micket!

**Pull request merged**